.github/workflows/build_upload_pypi_wheels.yml

```diff
@@ -38,14 +38,29 @@ jobs:
           python-version: ${{ matrix.python-version }}
           channels: conda-forge,defaults
           channel-priority: true
-      - name: Update Python pip, wheel, and twine
-        shell: bash -l {0}
-        run: |
-          python -m pip install --upgrade pip wheel twine
       - name: Install llvm on Macos
         if: startsWith(matrix.os, 'macos')
-        run: brew install llvm
-
+        shell: bash -l {0}
+        env:
+          # Homebrew location is different on Intel Mac
+          LLVM_DIR: ${{ (matrix.os == 'macos-13') && '/usr/local/opt/llvm' || '/opt/homebrew/opt/llvm' }}
+        run: |
+          brew install llvm
+          echo CC="${LLVM_DIR}/bin/clang" >> $GITHUB_ENV
+          echo LDFLAGS="-L${LLVM_DIR}/lib $LDFLAGS" >> $GITHUB_ENV
+          echo CPPFLAGS="-I${LLVM_DIR}/include $CPPFLAGS" >> $GITHUB_ENV
+      - name: Setup MSVC env on Windows
+        if: startsWith(matrix.os, 'windows')
+        shell: bash -l {0}
+        env:
+          MSVC_BIN: 'C:\Program Files\Microsoft Visual Studio\2022\Enterprise\VC\Tools\MSVC\14.41.34120\bin\HostX64\x64'
+        run: |
+          echo "CC=${MSVC_BIN}\cl.exe" >> $GITHUB_ENV
+          echo "CC_LD=${MSVC_BIN}\link.exe" >> $GITHUB_ENV
+      - name: Update Python pip, build, wheel, and twine
+        shell: bash -l {0}
+        run: |
+          python -m pip install --upgrade pip build wheel twine
       - name: Build Python wheel
 
         if: matrix.os != 'ubuntu-latest'
```

Comment:
Is there any reason why `cibuildwheel` isn't being used on other platforms?

Reply:
Excellent question. I never looked at this part closely, assumed it was just a tool that handles the "manylinux" complications.

Reply:
I note that the CIBW_REPAIR_WHEEL_COMMAND hook is intended for things like `delvewheel`, if we end up needing those particular Windows shenanigans.